oioioi/base/registration_backend.py

```diff
@@ -63,17 +63,23 @@ def register(self, form):
 
 
 urlpatterns = [
-    re_path(r'^register/$', RegistrationView.as_view(), name='registration_register'),
+    re_path(r'^sign-up/$', RegistrationView.as_view(), name='sign-up'),
+    re_path(r'^register/$', RedirectView.as_view(url='/sign-up/', permanent=True), name='registration_register'),
```

Comment:
Again, I am picky, but `registration_redirect` is so much better than `registration_register` in my opinion